hnn_core/network.py

```diff
@@ -410,9 +412,12 @@ def _create_all_spike_sources(self):
                             params=self.p_common[p_ind],
                             gid=gid))
 
-                # create the cell and artificial NetCon
-                pc.cell(gid, self.common_feeds[-1].connect_to_target(
-                        self.params['threshold']))
+                event_times = get_rhythmic_times(tstart=params['t0'],
+                                                 tstop=params['T'],
+                                                 ...)  # TODO: fill the rest
```

Comment:
@rythorpe I probably broke your code :) But I think I am getting a clearer and clearer picture now. I made a skeleton/sketch of what we want to accomplish in this PR. Please take a look and we'll discuss tomorrow. We really should explicitly pass parameters to make the code more transparent. If you get the idea already, feel free to pull the latest commit and continue on top of it. Let's catch up tomorrow in any case!